### ISSUE TITLE
Icons for provider type in "add host" screen not accessible

### ISSUE THREAD

**Issue by ndarilek** [rancher/rancher#1295]
As a blind screen reader user, I am unable to navigate the icons for provider types in the screen to add a host.

 * The icons are not keyboard-accessible. It's possible that `href="#"` is needed to enable this.
 * Similarly, they are completely invisible to my screen reader (NVDA, Windows 8.1) despite the presence of alt text. I only found them by cracking open Firebug. It's possible that you'll need an aria-label="..." attribute alongside alt="...", though the href also might do it.

Thanks.

**Comment by vincent99** [comment]
Fair enough.. We have not really tested accessibility at all so I expect there will be other problems as you get further in.  Please keep the issues coming.

**Comment by ndarilek** [comment]
Will do, thanks for the quick response! At the moment this has me
blocked since I'm running in a single-host Vagrant environment and can't
add the Vagrant instance as a custom host, but as soon as it's fixed
I'll be happy to poke things and give more feedback. :)

**Comment by ndarilek** [comment]
Possibly addressed in https://github.com/rancherio/ui/pull/217, sorry I filed this issue with the wrong repository.

Also would appreciate someone performing basic sanity checks on this PR. I'm on Windows so am likely limited on how much Rancher development I can do. But it literally just adds `href="#"` to the link, which I think should make them keyboard accessible. Let me know if it breaks something, and also please let me know when this fix makes it to Docker Hub. I'll pull it and test it in my local Vagrant environment.

Thanks!

**Comment by vincent99** [comment]
@ndarilek Your PR is in the `v0.28.0` release if you want to try again.  I also added a label to the "User Menu" which has the dropdown for API & Keys that you had asked about somewhere else.  There are some other buttons that I think are inaccessible that I will try go through for next release.  NVDA seems to be Windows-only, is there something roughly equivalent that I could try on a Mac (maybe just VoiceOver?)
